### DISCUSSION TITLE
Node.js version decision doc

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#259] [Announcements]
Hello folks! I just wrote a new decision document about the Node.js version we use: https://github.com/epicweb-dev/epic-stack/blob/main/docs/decisions/021-node-version.md

I also noticed that a new version of Debian has been released so I updated our base image to use `bookworm` as well.

Enjoy!

**Comment by tbroyer** [comment]
Fwiw, currently, Node 14 and 19 are EOL. 14 since late April, 19 since late June.

Apart from that, using LTS is the good decision; that's what Node is telling us to use.

**Reply by kentcdodds** [reply]
Whoops, you're right. I read the diagram wrong 😅 I thought something felt weird about it when I wrote it!
